components/Hobby.tsx

```
@@ -0,0 +1,25 @@
+  description?: string;
+  icon?: string;
+}
+
+const HobbyComponent: React.FC<HobbyProps> = ({ name, icon, description }) => {
+  return (
+    <div className="col">
+      <div className="card h-100">
+        {" "}
+        {/* Make cards equal height */}
+        <div className="card-body">
+          {icon && <i className={`${icon} fa-2x mb-2`}></i>}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)